nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -331,9 +331,9 @@ def create_client(self, client_name):
             client_name, self.args
         )
         self.federated_clients.append(client)
-        app_root = os.path.join(self.simulator_root, "app_" + client_name)
-        app_custom_folder = os.path.join(app_root, "custom")
-        sys.path.append(app_custom_folder)
+        # app_root = os.path.join(self.simulator_root, "app_" + client_name)
+        # app_custom_folder = os.path.join(app_root, "custom")
+        # sys.path.append(app_custom_folder)
```

Comment:
Where is the other place that we set this custom path for the clients?

Reply:
It's here:

https://github.com/NVIDIA/NVFlare/blob/main/nvflare/private/fed/app/simulator/simulator_worker.py#L149-L151